Update connection-pooling for Digital Ocean (#6462)
redwoodjs/redwood: docs/docs/connection-pooling.md

## Digital Ocean
For Postgres, see [How to Manage Connection Pools](https://www.digitalocean.com/docs/databases/postgresql/how-to/manage-connection-pools)

> In order to run migrations through a connection pool you are required to append additional connection parameters to your database url. Prisma needs to be informed to use pgbouncer (which is part of the connection pool of Digital Ocean), if not provided you may receive the following error:

Reviewer: ```suggestion
To run migrations through a connection pool, you're required to append connection parameters to your `DATABASE_URL`. Prisma needs to know to use pgbouncer (which is part of Digital Ocean's connection pool). If omitted, you may receive the following error:
```

> In order to run migrations on platforms like `Vercel` and `Netlify` you are required to append additional connection parameters to your database url. Prisma needs to be informed to use pgbouncer (which is part of the connection pool of Ditital Ocean), if not provided the following error will be show:

Reviewer 2: ```suggestion
In order to run migrations through a connection pool you are required to append additional connection parameters to your database url. Prisma needs to be informed to use pgbouncer (which is part of the connection pool of Digital Ocean), if not provided the following error will be show:
```

```
Error: Migration engine error:
db error: ERROR: prepared statement "s0" already exists
```

> To resolve this use the following structure of you database url: 

Reviewer: ```suggestion
To resolve this, use the following structure in your `DATABASE_URL`:

```

```
<YOUR_CONNECTION_POOL_URL>:25061/defaultdb?connection_limit=3&sslmode=require&pgbouncer=true&connect_timeout=10&pool_timeout=30
```

> Notes:

Reviewer: ```suggestion
Here's a couple more things to be aware of:
```

> * Make sure to use the correct host and  *NOT* `25060`

Author: Thanks for the feedback. I adjusted to text a little bit. Hopefully its more clear now :)

> - When using a connection pool of Digital Ocean you will have multiple port available.Typically the direct connection (without connection pooling) is on port 25060 and the connection through pgbouncer is served through port 25061. Make sure you connect to your connection pool on port **25061**

Reviewer: ```suggestion
- When using a Digital Ocean connection pool, you'll have multiple ports available. Typically the direct connection (without connection pooling) is on port `25060` and the connection through pgbouncer is served through port `25061`. Make sure you connect to your connection pool on port `25061`
```

> - Adjust the connection_limit: Clusters provide 25 connections per 1 GB of RAM. 3 connections per cluster are reserved for maintenance, and all remaining connections can be allocated to connection pools.

Reviewer: ```suggestion
- Adjust the `connection_limit`. Clusters provide 25 connections per 1 GB of RAM. Three connections per cluster are reserved for maintenance, and all remaining connections can be allocated to connection pools
```

> - Both `pgbouncer=true` and `pool_timeout=30` are required in order to deploy succesfully through your connection pool

Reviewer: ```suggestion
- Both `pgbouncer=true` and `pool_timeout=30` are required to deploy successfully through your connection pool
```
